packages/rfw/README.md

```diff
@@ -254,7 +776,7 @@ concerns on iOS, anyway).
 
 ## Contributing
 
-If you run into any problems, please file a [new bug](https://github.com/flutter/flutter/issues/new?labels=p:%20rfw,package,P4), though
+If you run into any problems, please file a [new bug](https://github.com/flutter/flutter/issues/new?labels=p:%20rfw,package,P2), though
 as noted above, you may have to fix the issue yourself and submit a PR.
```

Comment:
Was this noted above? Certainly it makes sense, but I'm not sure it was really stated anywhere above.

Reply:
that's just an update for https://flutter.dev/go/triage-2023-rfc

Reply:
Sure, but I meant the line after that link (in some text that hadn't changed, but I was reading the whole document anyway): "as noted above, you may have to fix the issue yourself and submit a PR."

With "as noted above," deleted, it would make perfect sense to me, but I don't see where it was noted above.

Reply:
oh heh, the note it's referring to was removed. i've removed this mention too.